### ISSUE TITLE
Semantic function of an adjunct

### ISSUE THREAD

**Issue by jonathanrobie** [biblicalhumanities/treedown#15]
Should we standardize an extension module for semantic function of an adjunct?

**Comment by jtauber** [comment]
That would be nice but it's a good test of the framework if this can be done independently and orthogonally.

**Comment by jonathanrobie** [comment]
Agreed.  In the examples I have given for this in other issues, I put this in a non-native namespace.  If we were to do something like this, we should do it as an external extension.
